climakitae/core/data_export.py

```diff
@@ -525,9 +622,15 @@ def _export_to_csv(data, output_path):
 
     _metadata_to_file(data, output_path)
     df.to_csv(output_path, compression="gzip")
+    print(
+        (
+            "Saved! You can find your file(s) in the panel to the left"
+            " and download to your local machine from there."
+        )
+    )
 
 
-def export(data, filename="dataexport", format="NetCDF"):
+def export(data, filename="dataexport", format="NetCDF", mode="auto"):
     """Save data as a file in the current working directory.
```

Comment:
or save the file to a S3 bucket and provide a
URL for download

Reply:
Added more documentation...